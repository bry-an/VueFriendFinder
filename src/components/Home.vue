
<template>
<div id='home-intro'>
<p>Welcome to Companion Finder!</p>
<p>This is a site made by an adult for adults.
    We have no preconceived notions about what makes for a good companion,
    only the simplest understanding about what it takes for a healthy relationship.</p>
<p>Fill out this quick form to discover your ideal companion -- and maybe even learn
    something about yourself. </p>
    <router-link to='/SurveyList' tag="button">Get started</router-link>
</div>
</template>

<script>
export default {
    name: "Home"
}
</script>
