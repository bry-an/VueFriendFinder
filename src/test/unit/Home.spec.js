import { shallowMount } from "@vue/test-utils"
import Home from "../../components/Home"

describe("Home", () => {
    it("is a vue instance", () => {
        const wrapper = shallowMount(Home)
        expect (wrapper.isVueInstance()).toBeTruthy()
    })
})