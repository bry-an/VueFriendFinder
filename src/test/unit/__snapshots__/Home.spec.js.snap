// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  id="home-intro"
>
  <p>
    Welcome to Companion Finder!
  </p>
   
  <p>
    This is a site made by an adult for adults.
    We have no preconceived notions about what makes for a good companion,
    only the simplest understanding about what it takes for a healthy relationship.
  </p>
   
  <p>
    Fill out this quick form to discover your ideal companion -- and maybe even learn
    something about yourself. 
  </p>
   
  <router-link-stub
    tag="button"
    to="/surveylist"
  >
    Get started
  </router-link-stub>
</div>
`;
